bundles/org.openhab.binding.mybmw/src/main/java/org/openhab/binding/mybmw/internal/MyBMWConstants.java

```diff
@@ -58,7 +58,7 @@ public interface MyBMWConstants {
 
     static final int DEFAULT_IMAGE_SIZE_PX = 1024;
 
-    static final int DEFAULT_REFRESH_INTERVAL_MINUTES = 5;
+    static final int DEFAULT_REFRESH_INTERVAL_MINUTES = 60;
```

Comment:
First of all, I'm not familiair with the API quota, so it might not make sense, but making it 12 times slower..

Reply:
well, statistically a car is not moving for a quite long time per day (I found one stating it is even moving only 15mins per day...), so I think it is sufficient to trigger the requests only once per hour. The parameter can easily be changed in the vehicle configuration, so I would rather set it lower than higher to avoid too high load on BMW API (and finally maybe to even get more throttling due to too high load on the API)